core/trino-main/src/main/java/io/trino/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -158,7 +160,7 @@ private AggregationAnalyzer(
             Metadata metadata,
             Analysis analysis)
     {
-        requireNonNull(groupByExpressions, "groupByExpressions is null");
+        this.groupByExpressions = requireNonNull(groupByExpressions, "groupByExpressions is null");
```

Comment:
1. defensive copy

2. as you probably noticed, the pattern in this method is that `requireNonNull`-s come first, then assignments. Any reason not to remain consistent?